kubernetes_asyncio/__init__.py

```diff
@@ -20,3 +20,4 @@
 import kubernetes_asyncio.config
 import kubernetes_asyncio.watch
 import kubernetes_asyncio.stream
+import kubernetes_asyncio.utils
```

Comment:
Could you add this module to the coverage report?

https://github.com/tomplus/kubernetes_asyncio/blob/594a639e2e202f20fb3700d014593bde26cdd15f/tox.ini#L40

Reply:
ok